edm4eic.yaml

```
@@ -203,6 +203,27 @@ datatypes:
       - float             floatData       // Iff elementType==1, values are stored here
       - int64_t           int64Data       // Iff elementType==7, values are stored here
 
+  ## ==========================================================================
+  ## Simulation info
+  ## ==========================================================================
+
+  edm4eic::SimPulse:
+    Description: "Simulated pulse prior to digitization."
+    Author: "D. Anderson, S. Gardner, S. Joosten., D. Kalinkin"
+    Members:
+      - uint64_t            cellID          // ID of the readout cell for this pulse.
+      - float               integral        // Total pulse integral in relevant units.
+      - edm4hep::Vector3f   position        // Position of the hit in world coordinates [mm].
```

```suggestion
      + edm4hep::Vector3f   position        // Position the pulse is evaluated in world coordinates [mm].
```
Or something similar to add clarity.